internal/controllers/gateway/utils.go

```diff
@@ -0,0 +1,23 @@
+package gateway
+
+import (
+	"github.com/go-logr/logr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kong/kubernetes-ingress-controller/v2/internal/util"
+)
+
+// -----------------------------------------------------------------------------
+// Logging Utilities
+// -----------------------------------------------------------------------------
+
+// debug is an alias for the longer log.V(debugLevel).Info for convenience
+func debug(log logr.Logger, obj client.Object, msg string, keysAndValues ...interface{}) {
+	debugLevel := util.InfoLevel // temporarily upgrading all debug to info while developing
```

Comment:
This was already covered by https://github.com/Kong/kubernetes-ingress-controller/issues/1988 so that it wouldn't be forgotten, but now I added it to the comment in eda82b7f62d2d7222954ddda8a3276f3cae37a6e